runtime/src/mailbox_api.rs

```diff
@@ -65,7 +63,7 @@ impl MailboxResp {
             MailboxResp::GetIdevCsr(resp) => resp.as_bytes(),
             MailboxResp::GetLdevCert(resp) => resp.as_bytes(),
             MailboxResp::StashMeasurement(resp) => resp.as_bytes(),
-            MailboxResp::InvokeDpeCommand(resp) => resp.as_bytes(),
+            MailboxResp::InvokeDpeCommand(resp) => resp.as_bytes_partial(),
```

Comment:
Should we cover the as_bytes_mut case too? I think it's only currently used in the populate_chksum function in this file and it doesn't matter there technically.

Reply:
Ah, ya I'll update it there as well just to be sure. It is interesting though because the checksum will be the same as long as all the trailing bytes are 0 😄

Reply:
Yeah I can't think of a scenario where it would actually break anything since we use "as_bytes" for the mailbox send call. Just a question of covering it for completeness. I wish there were a cleaner way to implement functions like this that only differ in mutability.